src/app/shared/components/search/search.component.html

```diff
@@ -1,8 +1,16 @@
 <div class="search">
   <div>
-    <span class="img-search"><img src="assets/img/ic-search@2x.png" class="ic_search"></span>
-    <input type="search" placeholder="Text to search"/>
-    <button type="button" class="btn btn-default sick-green">Search</button>
+    <span class="img-search"
+      ><img src="assets/img/ic-search@2x.png" class="ic_search"
+    /></span>
+    <input #searchInput type="search" placeholder="Text to search" />
+    <button
+      type="button"
+      class="btn btn-default sick-green"
+      (click)="find(searchInput.value)"
+    >
```

Comment:
The arrow ">" should be on the previous line, like this it just takes extra line in the code.